src/docker-utils.coffee

```diff
@@ -206,6 +274,6 @@ do ->
 						console.log 'Progress error: Unknown layer ' + id + ' downloaded by docker. Progress not correct.'
 						totalSize = null
 				downloadedSize = completedSize + currentSize
-				percentage = calculatePercentage(downloadedSize, totalSize)
+				percent = calculatePercentage(downloadedSize, totalSize)
 
-				onProgress({ downloadedSize, totalSize, percentage })
+				onProgress({ downloadedSize, totalSize, percent })
```

Comment:
I submitted the PR for that here: https://github.com/resin-io/resin-supervisor/pull/10/files - I already had the majority of the work locally as I did it as part of creating the docker-progress module